contents.json

```diff
@@ -17039,6 +17039,25 @@
       ],
       "date_added": "Sep 3 2018",
       "suggested_by": "@vpeschenkov"
+    },
+    {
+      "title": "Good Spirits",
+      "category-ids": [
+        "health"
+      ],
+      "description": "A drink tracker with weekly limits, Untappd support, and HealthKit sync.",
+      "source": "https://github.com/archagon/good-spirits",
```

Comment:
@archagon Could you add the license of this source, like this `"license": "gpl-3.0"`